charts/ndb-operator/templates/apps_v1_deployment_controller-manager.yaml

```diff
@@ -1,89 +0,0 @@
-apiVersion: apps/v1
-kind: Deployment
-metadata:
```

Comment:
Note: This file has changed a lot. Please review this file carefully. 
The new version of this file is "deployment-ndb-operator-controller-manager.yaml"